tests/config/test_themes.py

```diff
@@ -42,6 +71,37 @@ def test_builtin_theme_completeness(theme_name):
     assert all(required_style in styles_in_theme for required_style in REQUIRED_STYLES)
 
 
+@pytest.mark.parametrize(
+    "theme_name, depth",
+    [
+        ("zt_dark", 16),
```

Comment:
You've not tested the monochrome bit depth? While that data hasn't been updated, we're migrating from the old function to using generate_theme, so it would be good to include here.

Reply:
Shouldn't the REQUIRED_STYLES completeness tests be enough because this would just compare REQUIRED_STYLES with REQUIRED_STYLES? It wouldn't actually be testing anything right? It would never fail as well - I think?

Reply:
This isn't critical, but seemed missing from the set. I understand it seems "obvious", but many automated tests could be considered that way - though this one more than others perhaps.

Reply:
I've added this check as well